Source/Serilog.Exceptions/Reflection/ReflectionInfo.cs

```diff
@@ -0,0 +1,15 @@
+namespace Serilog.Exceptions.Reflection
+{
+    internal class ReflectionInfo
```

Comment:
Can we use a `record` type for this now?

Reply:
I think I can, but I do not assign and equality semantics to this class at all, especially not value-based semantics, so I assume it does not matter.

Reply:
Actually, it would be my first record created, so I may also have no best practices in my blood yet. If you do not mind, please choose whether to use a record or not here.

Reply:
I think it's only available in new TFM's so we can't use it everywhere.